Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -456,6 +456,17 @@ def build_query_for_request_params(query):
         return query
 
 
+def escape_build_query(sys_param_query, escape_amp=True):
+    """
+      Used to escape or build the query parameters
+    """
+
+    if sys_param_query:
+        if escape_amp:
+            return build_query_for_request_params(sys_param_query)
```

Comment:
not sure I understand.
`build_query_for_request_params` is doing `parse.parse_qsl`, to "escaping". so where does the escaping comes in to it?

Reply:
`build_query_for_request_params` is the function that caused the bug. Here I test if we want to execute it or not.